preprocessors/clothes.schema.json

```diff
@@ -0,0 +1,47 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/clothes-detector.schema.json",
+    "type": "object",
+    "title": "Clothes Detector",
+    "description": "Preprocessor to detect the clothes worn by the individuals. Since clothes detection should work only on people, we would add a condition in the associated preprocessor where the code would be run only on people detected by the object detector.",
+    "properties": {
+        "clothes": {
+            "type": "array",
+            "description": "Contains a list of clothes worn by people",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "personID": {
+                        "description": "The ID of a person. The ID is generated by object detection preprocessor and is reused in this preprocessor",
+                        "type": "integer",
+                        "minimum": 0
+                    },
+                    "attire": {
+                        "type": "array",
+                        "description": "Lists the different clothes worn by an individual. Since one individual can wear multiple clothes such as a short sleeve top and an outerwear, we have used an array to describe the entire attire of the individual. Each element of an array would describe one piece of cloth worn by the individual.",
+                        "items": {
+                            "type": "object",
+                            "properties": {
+                                "clothes": {
```

Comment:
Perhaps: article? type? What kind of values does this actually take? Is it something as simple as "t-shirt" or is this a full description like "a blue t-shirt with a green pocket"?

Reply:
so this would just generate t-shirt, dress, skirt.